### PR TITLE
[release-0.18] Add the missing 0.18.6 eventing manifests

### PR DESCRIPTION
**Pull request by houshengbo**
## Proposed Changes

* Add the missing 0.18.6 eventing manifests

### CODE REVIEW

File: .github/workflows/knative-boilerplate.yaml [hunk 1]
```diff
@@ -64,7 +64,7 @@ jobs:
           go get github.com/mattmoor/boilerplate-check/cmd/boilerplate-check
           echo '::endgroup::'
 
-          echo "::add-path::${TEMP_PATH}"
+          echo "${TEMP_PATH}" >> $GITHUB_PATH
```
Comment (antoineco) [inline .github/workflows/knative-boilerplate.yaml:67]: Is that a leftover from some automated update?

File: .github/workflows/knative-style.yaml [hunk 2]
```diff
@@ -109,7 +109,7 @@ jobs:
           curl -sfL https://raw.githubusercontent.com/get-woke/woke/main/install.sh | sh -s -- -b "${TEMP_PATH}" "${WOKE_VERSION}" 2>&1
           echo '::endgroup::'
 
-          echo "::add-path::${TEMP_PATH}"
+          echo "${TEMP_PATH}" >> $GITHUB_PATH
```
Comment (antoineco) [inline .github/workflows/knative-style.yaml:112]: Same here?

File: .github/workflows/knative-boilerplate.yaml (hunk 1 above)
Comment (houshengbo) [reply .github/workflows/knative-boilerplate.yaml:67]: Correct it fixes the github action failures.

File: .github/workflows/knative-style.yaml (hunk 2 above)
Comment (houshengbo) [reply .github/workflows/knative-style.yaml:112]: Correct it fixes the github action failures.

Comment (houshengbo) [reply .github/workflows/knative-style.yaml:112]: https://github.com/knative/serving/commit/d2ec5468b2fd5c018d314eba24941e3f22331264 We need to do the same as in here.

Comment (antoineco) [reply .github/workflows/knative-style.yaml:112]: Got it 👍 

/lgtm

**Pull request merged**